packages/react-scripts/config/webpack.config.js

```diff
@@ -250,16 +229,6 @@ module.exports = function(webpackEnv) {
           },
         }),
       ],
-      // Automatically split vendor and commons
-      // https://twitter.com/wSokra/status/969633336732905474
-      // https://medium.com/webpack/webpack-4-code-splitting-chunk-graph-and-the-splitchunks-optimization-be739a861366
-      splitChunks: {
-        chunks: 'all',
-        name: false,
-      },
-      // Keep the runtime chunk separated to enable long term caching
-      // https://twitter.com/wSokra/status/969679223278505985
-      runtimeChunk: true,
```

Comment:
Chose not to code split because I was having problems of loading hashed bundles.

Caches are no longer valuable since extensions comes with the JS saved local to user browsers. We do unfortunately lose the performance gains of parsing js code quicker.